Jenkinsfile

```diff
@@ -158,7 +159,7 @@ def buildUnitTestStage(env) {
     stage("Run ${env} unit tests") {
       container("maven-${env}") {
         script {
-          setGitHubBuildStatus("platform/utests/${env}", "Unit tests - ${env} environment", 'PENDING')
+          setGitHubBuildStatus("${kubernetesNamespace}/utests/${env}", "Unit tests - ${env} environment", 'PENDING')
```

Comment:
I would say this one is not linked to the Kubernetes namespace, it's just a GitHub status name.

Reply:
yes I reverted all those gh status changes